cli/uninstall/uninstall.go

```diff
@@ -112,12 +112,30 @@ func UninstallProgram(program string, programVersion string, binDst string, head
 			return err
 		}
 	}
+	// We need to try to delete all versions with 'v' prefix before x.y.z version
+	// and without it.
+	versionsToDelete := make([]string, 0, 2)
+	versionsToDelete = append(versionsToDelete, programVersion)
+
+	if program == search.ProgramTt &&
+		strings.Contains(programVersion, ".") && programVersion[0] != 'v' {
+		versionsToDelete = append(versionsToDelete, "v"+programVersion)
```

Comment:
Please, add a comment why we need the logic here.

Reply:
Added.